cypress/integration/seededFlows/adminFlows/users/manageOrganizations.spec.js

```diff
@@ -0,0 +1,134 @@
+// More on roles, https://admin.forem.com/docs/forem-basics/user-roles
+function openOrgModal(ctaText = 'Add organization') {
+  cy.getModal().should('not.exist');
+  cy.findByRole('button', { name: ctaText }).click();
+
+  return cy.getModal();
+}
+
+function closeUserUpdatedMessage(message) {
+  cy.findByText(message).should('exist');
+  cy.findByRole('button', { name: 'Close' }).click();
+  cy.findByText(message).should('not.exist');
+}
+
+describe('Manage User Organziations', () => {
+  describe('As an admin', () => {
+    beforeEach(() => {
+      cy.testSetup();
+      cy.fixture('users/adminUser.json').as('user');
+      cy.get('@user').then((user) => {
+        cy.loginUser(user);
+      });
+    });
+
+    it(`should add a user to an organization`, () => {
+      cy.visit('/admin/users/3');
+
+      cy.findByText('This user is not a part of any organization yet.').should(
+        'be.visible',
+      );
+
+      openOrgModal().within(() => {
+        cy.findByRole('spinbutton', { name: 'Organization ID' }).type(1);
+        cy.findByRole('button', { name: 'Add organization' }).click();
+      });
+
+      closeUserUpdatedMessage('User was successfully added to Bachmanity');
+      cy.getModal().should('not.exist');
+
+      cy.findByRole('link', { name: 'Bachmanity' }).focus();
```

Comment:
Is this so that the button becomes visible for clicking? Just thinking maybe a comment around that would help anyone looking at this in the future